src/click/utils.py

```diff
@@ -29,6 +30,46 @@
 R = t.TypeVar("R")
 
 
```

Comment:
Move to _utils.py. Remove __bool__.

Reply:
`__bool__` removed in: https://github.com/pallets/click/pull/3030/commits/f4b5b5219759fe74826d9df48bd16449d17bc466

`UNSET` moved to `_utils` in: https://github.com/pallets/click/pull/3030/commits/d8c2d433d66c0cf434384128c4b38fca03e23718